op-node/node/api.go

```diff
@@ -163,6 +167,13 @@ func (n *nodeAPI) RollupConfig(_ context.Context) (*rollup.Config, error) {
 	return n.config, nil
 }
 
+func (n *nodeAPI) DependencySet(_ context.Context) (depset.DependencySet, error) {
```

Comment:
I assume `depset.DependencySet` needs to be serializable to json?  I don't see the serialization logic anywhere.